.github/workflows/post-dependabot.yml

```diff
@@ -0,0 +1,43 @@
+name: Dependabot post-update (rebuild moderator model)
+on:
+  pull_request_target:
+    action: [opened, synchronize, reopened]
+    branches:
+      - 'master'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+permissions: {}
+jobs:
+  post-update:
+    permissions:
+      pull-requests: read # for gh pr checkout
+      contents: write # to push code in repo (stefanzweifel/git-auto-commit-action)
+
+    if: github.actor == 'dependabot[bot]' && contains(github.event.pull_request.title, 'scikit-learn')
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Login
+      run: |
+        echo "${{ secrets.GITHUB_TOKEN }}" | gh auth login --with-token
+    - name: Checkout
+      run: |
+        gh pr checkout ${{ github.event.pull_request.number }}
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      cache: 'pip'
+      cache-dependency-path: '**/requirements*.txt'
+    - name: Download train.csv
+      run: curl "$MODERATOR_TRAINING_URL" > train.csv
```

Comment:
What's this secret? Why not have it in the open?

Reply:
Explained in the PR description.

Reply:
Ah, okay. Marking as resolved.